multicluster/apis/multicluster/v1alpha1/clusterclaim_types.go

```diff
@@ -30,7 +30,9 @@ const (
 // +genclient
 //+kubebuilder:object:root=true
 
-// ClusterClaim is the Schema for the clusterclaims API
+// +kubebuilder:printcolumn:name="Value",type=string,JSONPath=`.value`,description="Value of the ClusterClaim"
+// +kubebuilder:printcolumn:name="Age",type="date",JSONPath=".metadata.creationTimestamp"
```

Comment:
```suggestion
// +kubebuilder:printcolumn:name="Age",type="date",JSONPath=`.metadata.creationTimestamp`
```
looks like both \`\` and `""` works, we may keep the format consistent.
There are a few other comments need to be updated.

Reply:
Sure, updated.